symLogging.py

```diff
@@ -19,10 +18,11 @@ def mkdir_p(path):
 def SetLoggingOptions(logOptions):
   global gLog
 
+  gLog = logging.getLogger("tornado.application")
```

Comment:
No, this is for the application usage. I changed it because tornado already appends a `StreamHandler` to the logger, which caused duplicate messages on screeen.